examples/cusparselt/matmul.py

```diff
@@ -92,5 +92,8 @@
 #
 # destroys plan and handle
 #
+cusparselt.matDescriptorDestroy(matA)
+cusparselt.matDescriptorDestroy(matB)
+cusparselt.matDescriptorDestroy(matC)
```

Comment:
These would not work if we return `CUSPARSE_STATUS_NOT_SUPPORTED` for cuSPARSELt v0.0.1, so this may need to change depending on the final decision.